server.js

```diff
@@ -1,15 +1,37 @@
-const express = require('express')
-const { indexHandler, sayHelloHandler } = require('./routes/handlers')
-
+const express = require("express");
+const pino = require("pino");
+const { indexHandler, sayHelloHandler } = require("./routes/handlers");
 
 const app = express();
 const port = 3000;
 
+const logger = pino({
+  transport: {
+    target: "pino-pretty",
+    options: {
+      translateTime: "SYS:dd-mm-yyyy HH:MM:ss", // formate the date and hour
+      ignore: "pino,hostname", // remove the machina name (marina-ubuntu)
+    },
+  },
+});
 
-app.get('/', indexHandler);
+app.get("/", (req, res) => {
+  logger.info("info: the index page has been visited");
```

Comment:
Not necessary to prefix your message with **info:** as the log level has been provided by `logger.info()`